packages/okta-vue/test/e2e/harness/package.json

```diff
@@ -6,11 +6,12 @@
   "private": true,
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
+    "kill:port": "kill -s TERM $(lsof -t -i:8080 -sTCP:LISTEN) || true",
     "start": "yarn env && yarn dev",
     "env": "node ./scripts/prebuild.js",
     "pree2e": "../../../../../scripts/updateSeDrivers.sh",
     "e2e": "node test/e2e/runner.js",
-    "test": "yarn env && yarn e2e",
+    "test": "yarn env && yarn e2e && yarn kill:port",
```

Comment:
If the tests fail kill:port won't run, correct?

Reply:
Good observation. `posttest` might be a better place for this